csds-client/client/clientV2.go

```diff
@@ -0,0 +1,289 @@
+package client
```

Comment:
What do you think introducing a new package (either through naming, or through the package path like "v2/client.go" for each new version? This way you will avoid naming collisions between versions and wouldn't need to use names like "functionName_v2".

If you use different packages - code separation will become completely clean.

Reply:
I'm not sure if I got the point. I added a subdirectory `V2` under the `client` and moved `clientV2.go` to the package `clientV2`.
For the helper functions which are not related to api version, I kept them in `util.go` and exported them for `clientV2` to call and use them.